docs/guides/middleware.md

```diff
@@ -5,28 +5,43 @@ Middleware is a Video.js feature that allows interaction with and modification o
 ## Table of Contents
 
 * [Understanding Middleware](#understanding-middleware)
+  * [setSource](#setsource)
+  * [setTech](#setTech)
   * [Middleware Setters](#middleware-setters)
   * [Middleware Getters](#middleware-getters)
   * [Middleware Mediators](#middleware-mediators)
   * [Termination and Mediators](#termination-and-mediators)
 * [Using Middleware](#using-middleware)
   * [Terminating Mediator Methods](#terminating-mediator-methods)
-* [setSource](#setsource)
 
 ## Understanding Middleware
```

Comment:
I suggest starting out with a purpose statement for readers who have arrived at this page with no context. Something like "Middleware allows you to override the behavior of basic player methods."

Reply:
That's actually at the top of the file 😄

Reply:
Doh, missed that. Looks good!